src/platform/forms-system/test/js/review/SubmitController.unit.spec.jsx

```diff
@@ -109,12 +109,10 @@ const createStore = (options = {}) => {
 };
 
 describe('Schemaform review: SubmitController', () => {
-  beforeEach(() => {
-    testkit.reset();
-  });
-  afterEach(() => {
+  before(() => {
```

Comment:
I added a [comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/27231#issuecomment-881804204) about why this change is needed in the original ticket.

Reply:
Like this, we're implicitly trusting that there's only one test in this `describe` block that uses `testkit`. :thinking: And if the problem is resetting it when it hasn't been used, I'm not sure that this won't run into issues later.

But then again, I don't expect we'll do much work on this before it's deprecated officially, so :man_shrugging: I guess.

Reply:
I completely agree, just not sure why it isn't working with the Sentry upgrade in this instance 🤔. If you do end up working on it and can get it working in the `beforeEach` and `afterEach` hooks, that'd be great!